### PR TITLE
update cedar lang versions

### PR DESCRIPTION
**Pull request by khieta**
Updates to match [cedar#1019](https://github.com/cedar-policy/cedar/pull/1019)

### CODE REVIEW

File: docs/collections/_other/doc-history.md [hunk 1]
```diff
@@ -7,28 +7,29 @@ nav_order: 14
...
+
+| Cedar<br/>Version | Description | Cedar SDK<br/>Version(s) | Date |
+| --- | --- | --- | --- |
+| 3.3 | References between common types ([issue#154](https://github.com/cedar-policy/cedar/issues/154)) | 3.2.0 - 3.2.1 | May 17, 2024 |
+| 3.2 | General multiplication operator ([rfc#57](https://github.com/cedar-policy/rfcs/blob/main/text/0057-general-multiplication.md)) | 3.1.2 - 3.1.4 | March 29, 2024 |
+| 3.1 | New schema syntax ([rfc#24](https://github.com/cedar-policy/rfcs/blob/main/text/0024-schema-syntax.md)) | 3.1.0 - 3.1.1 | March 8, 2024 |
+| 3.0 | `is` operator ([rfc#5](https://github.com/cedar-policy/rfcs/blob/main/text/0005-is-operator.md))<br/>Stricter validation ([rfc#19](https://github.com/cedar-policy/rfcs/blob/main/text/0019-stricter-validation.md))<br/>Disallow duplicate keys in records ([rfc#20](https://github.com/cedar-policy/rfcs/blob/main/text/0020-unique-record-keys.md))<br/>Request validation ([issue#191](https://github.com/cedar-policy/cedar/issues/191)) | 3.0.0 - 3.0.1 | December 15, 2023 |
```
Comment (cdisselkoen) [inline docs/collections/_other/doc-history.md:32]: If we include request validation, should we also include entity data validation?

**Review by cdisselkoen — APPROVED** [review]

File: docs/collections/_other/doc-history.md (hunk 1 above)
Comment (khieta) [reply docs/collections/_other/doc-history.md]: Makes sense. Added in the latest commit.

**Review by cdisselkoen — APPROVED** [review]

File: docs/collections/_other/doc-history.md [hunk 2]
```diff
@@ -7,28 +7,29 @@ nav_order: 14
...
+| 3.3 | References between common types ([cedar#154](https://github.com/cedar-policy/cedar/issues/154)) | 3.2.0 - 3.2.1 | May 17, 2024 |
+| 3.2 | General multiplication operator ([rfc#57](https://github.com/cedar-policy/rfcs/blob/main/text/0057-general-multiplication.md)) | 3.1.2 - 3.1.4 | March 29, 2024 |
+| 3.1 | New schema syntax ([rfc#24](https://github.com/cedar-policy/rfcs/blob/main/text/0024-schema-syntax.md)) | 3.1.0 - 3.1.1 | March 8, 2024 |
+| 3.0 | `is` operator ([rfc#5](https://github.com/cedar-policy/rfcs/blob/main/text/0005-is-operator.md))<br/>Stricter validation ([rfc#19](https://github.com/cedar-policy/rfcs/blob/main/text/0019-stricter-validation.md))<br/>Disallow duplicate keys in records ([rfc#20](https://github.com/cedar-policy/rfcs/blob/main/text/0020-unique-record-keys.md))<br/>Request validation ([cedar#191](https://github.com/cedar-policy/cedar/issues/191))<br/>Entity validation ([cedar#360](https://github.com/cedar-policy/cedar/pull/360)) | 3.0.0 - 3.0.1 | December 15, 2023 |
+| 2.2 | General multiplication operator ([rfc#57](https://github.com/cedar-policy/rfcs/blob/main/text/0057-general-multiplication.md)) | 2.4.5 - 2.4.7 | April 1, 2024 |
```
[the remark this answers is not in the document]
Comment (cdisselkoen) [reply docs/collections/_other/doc-history.md:33]: It is the same change, applied to 2.x and to 3.x

[the remark this answers is not in the document]
Comment (khieta) [reply docs/collections/_other/doc-history.md]: Added "(backport to 2.x)". Here's the current table (rendered directly from markdown -- webpage formatting may differ):
Screenshot 2024-06-27 at 2 22 48 PM

**Review by shaobo-he-aws — APPROVED** [review]

**Pull request merged**